### ISSUE TITLE
Does not syncronize correctly

### ISSUE THREAD

**Issue by xadox-1st** [razzeee/script.trakt#458]
I completely reinstalled Kodi 18.1 (LibreElec 9.0.1) on a Raspberry Pi 2.
When I now resynchronize with Trakt, some already seen series episodes are not marked as seen.

On the Trakt website the series episodes are shown as seen.
Only the plugin doesn't do this for Kodi.

One of the series is e.g. [_The Animatrix_](https://trakt.tv/shows/the-animatrix). On the Trakt website I saw it completely and in Kodi it is shown as unseen.

**Comment by xadox-1st** [comment]
Reverting back to Kodi 17.6 (LibreElec 8.2.5) solved the problem.

**Comment by the-lazy-fox** [comment]
Which mean Leia incompatible?
Any idea when it will be fully compatible?

**Comment by razzeee** [comment]
If you upgrade from leia everything should be fine.

If you setup a new instance of kodi, make sure to change you show scraper to thetvdb for now. I think that's causing the problems.

**Comment by UncleStark** [comment]
When use plex and plexkodiconnect we can not specify scrappers. in 17.6 all working, in 18.2 movies scrobble sucsessifully, tvshows not scrobbles

**Comment by razzeee** [comment]
The scraper shouldn't be the important part, plexkodiconnect probably needs to talk the new format.

**Comment by UncleStark** [comment]
sync doesn't work either. I experimented and realized that the problem is not synchronization itself. problem in the names of the show. if they are not in English, then nothing works. but at 17.6 everything worked smoothly, stopped only in 18

**Comment by razzeee** [comment]
names do not matter if the ids are correct. and that's what i was hinting at with the last comment. they need to fix the format they are using, as the id structure changed.

**Comment by UncleStark** [comment]
Sorry if I look overly annoying. but why if rename the show to English, then everything starts working in the same setup

**Comment by razzeee** [comment]
Workflow:
1. try if you can match the show by id
2. if you could not and it's activated in the settings, try to match by name.
3. nothing worked? fail

So for your non english show you end up at 3, as the ids no longer work. But if we can rely on the name you end up at 2.

**Comment by UncleStark** [comment]
It turns out to solve the problem me need to contact the developer plexkodiconnect?

**Comment by razzeee** [comment]
Yes, this code is the relevant change on our side https://github.com/trakt/script.trakt/commit/87a7cb9882ab9756959146dda72357599d735435

**Comment by SonOfDiablo** [comment]
I don't know if this has anything to do with the problems people who have written in this issue has, but I just had some problems with an episode that wouldn't sync properly (it was unwatched in Kodi, but watched on Trakt).
After some fiddling I ended up looking in the  MyVideos database and saw the episode showed up twice. It turned out I had a duplicate of the episode file, Kodi imported both, but the Trakt script only handled one of them.

The "uniqueid_value" field was the same on both which corresponds to the correct tvdb ID.
